app/scripts/modules/core/pipeline/config/stages/tagImage/tagImageStage.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.stage.manualJudgmentStage', [])
```

Comment:
Doh. Hah. Fixed. Should have double checked everything rather than pushing a late night PR.